igcollect/artfiles.py

```diff
@@ -31,21 +37,23 @@ def parse_args():
 
 def main():
     args = parse_args()
-    artfiles_url = (
-        'https://{}:{}@dcp.c.artfiles.de/api/stats/get_estats.html'
-        .format(args.http_user, args.http_pass)
-    )
-    data = urlopen(artfiles_url)
+    request = Request('https://dcp.c.artfiles.de/api/stats/get_estats.html')
+    credentials = '{}:{}'.format(args.http_user, args.http_pass)
+    base64credentials = base64.b64encode(credentials.encode())
+    request.add_header('Authorization',
```

Comment:
Why aren't you using the `auth` parameter of the constructor of Request?
https://github.com/kennethreitz/requests/blob/master/requests/models.py#L214

Reply:
Ah, nevermind, you are using the urllib module not the request module. But then, one could consider to use the request module. ^^

Reply:
I used urllib to avoid having external dependencies.

Reply:
Quite sure, that we kinda habe the request module in every place, as it is so common. But you are right, that we do not need it. Keep it like that. Good to go like that.